src/current/v23.2/upgrade-cockroach-version.md

```diff
@@ -193,6 +189,8 @@ These steps perform an upgrade to the latest {{ page.version.version }} release,
     cp -i {COCKROACHDB_DIR}/cockroach /usr/local/bin/cockroach
     ~~~
 
+1. If a cluster has corrupt descriptors, a major-version upgrade cannot be finalized. In CockroachDB v23.2 and above, automatic descriptor repair is enabled by default. After restarting each cluster node on v23.2, monitor the cluster logs for errors. If a descriptor cannot be repaired automatically, [contact support](https://support.cockroachlabs.com/hc) for assistance completing the upgrade. To disable automatic descriptor repair (not generally recommended), set the environment variable `COCKROACH_RUN_FIRST_UPGRADE_PRECONDITION` to `false`.
```

Comment:
```suggestion
1. If a cluster has corrupt descriptors, a major-version upgrade cannot be finalized. In CockroachDB v23.2 and above, automatic descriptor repair is enabled by default. After restarting each cluster node on v23.2, monitor the [cluster logs](https://www.cockroachlabs.com/docs/{{ page.version.version }}/logging) for errors. If a descriptor cannot be repaired automatically, [contact support](https://support.cockroachlabs.com/hc) for assistance completing the upgrade. To disable automatic descriptor repair (not generally recommended), set the environment variable `COCKROACH_RUN_FIRST_UPGRADE_PRECONDITION` to `false`.
```
I won't update the support link now, because it's the same link used widely throughout the docs (and in the Support section of the footer, for example). If they ever change the endpoint, they will need to figure out a way to redirect, I imagine.

I see what you mean about "generally" seeming extraneous, but if there is a bug that prevents the upgrade from completing, support probably will recommend that a specific customer disable it. So I think I will leave that word there for now, to cover us.